lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4261,6 +4275,9 @@ EffectiveClangContext ClangImporter::Implementation::getEffectiveClangContext(
   // effective Clang context.
   if (auto constClangDecl = nominal->getClangDecl()) {
     auto clangDecl = const_cast<clang::Decl *>(constClangDecl);
+    if (isa<clang::NamespaceDecl>(clangDecl))
```

Comment:
If this API is used elsewhere, the behavior might be a bit odd. What do you think about passing `None` as the second argument to `lookupTable->lookup` on L4074 instead?

Reply:
My original implementation did that, so I can just go back to that.

Reply:
Perfect. I think this might actually be faster too, because this way we don't filter twice.

Reply:
(But I suppose we have a bigger vector, so maybe not...)